build.cmd

```diff
@@ -61,11 +61,6 @@ if "%__buildSpec%"=="managed"  goto :BuildManaged
 :: Run the Native Windows build
 echo [%time%] Building Native Libraries...
 :: Generate Native versioning assets
```

Comment:
nit: If you are removing the code you can remove the comment above as well.

@joperezr is this version generation no longer needed?